src/components/engagement/dto/create-engagement.dto.ts

```diff
@@ -55,6 +55,9 @@ export abstract class CreateLanguageEngagement extends CreateEngagement {
 
   @Field({ nullable: true })
   readonly historicGoal?: string;
+
+  @Field({ nullable: true })
+  readonly approvedInventory?: boolean;
```

Comment:
Do we need to expose this field or create/update only in the handler?

Reply:
It can be on create and update